chalice/pipeline.py

```diff
@@ -9,18 +9,36 @@
 def create_pipeline_template(config):
     # type: (Config) -> Dict[str, Any]
     pipeline = CreatePipelineTemplate()
-    return pipeline.create_template(config.app_name)
+    return pipeline.create_template(config.app_name,
+                                    config.lambda_python_version)
+
+
+class InvalidCodeBuildPythonVersion(Exception):
+    def __init__(self, version):
+        # type: (str) -> None
+        super(InvalidCodeBuildPythonVersion, self).__init__(
+            'CodeBuild does not yet support python version %s.' % version
+        )
 
 
 class CreatePipelineTemplate(object):
 
+    _CODEBUILD_IMAGE = {
+        'python2.7': 'python:2.7.12',
+    }
+
     _BASE_TEMPLATE = {
         "AWSTemplateFormatVersion": "2010-09-09",
         "Parameters": {
             "ApplicationName": {
                 "Default": "ChaliceApp",
                 "Type": "String",
                 "Description": "Enter the name of your application"
+            },
+            "CodeBuildImage": {
+                "Default": "python:2.7.12",
```

Comment:
Don't you need to change this default to `aws/codebuild/python:2.7.12`  now that the `aws/codebuild` isn't used in the template?